pkg/ipc/ipc.go

```diff
@@ -52,8 +52,8 @@ const (
 	FlagDedupCover                                 // deduplicate coverage in executor
 	FlagCollectComps                               // collect KCOV comparisons
 	FlagThreaded                                   // use multiple threads to mitigate blocked syscalls
-	FlagCollide                                    // collide syscalls to provoke data races
 	FlagEnableCoverageFilter                       // setup and use bitmap to do coverage filter
+	FlagCollectSignal
```

Comment:
Please make this the first flag. Logically it needs to be grouped with FlagCollectCover, and I think it makes it to place as the first one because logically it's the most important and the most used of all coverage/KCOV-related flags.
And add a comment.

Reply:
Done. Btw, how do you think, is it reasonable to make it set by default here?

https://github.com/google/syzkaller/blob/master/pkg/ipc/ipcconfig/ipcconfig.go#L43

Reply:
Probably (if we set ipc.FlagSignal).